docs/kb/api-version-4/prospects.md

```diff
@@ -269,11 +267,11 @@ JSON Example:
             "first_name": "New first name",
             "last_name": "New last name"
         },
-        "some@email.com": {
+        "1337": {
```

Comment:
Actually, you need to have a unique identifier, so I think it has to look like this: 
```json
{
    "prospects": {
        "1":{"email":"some@email.com","first_name":"First name","last_name":"Last name"},
        "2":{"email":"some@email.com","first_name":"First name","last_name":"Last name"},
        "3":{"email":"some@email.com","first_name":"First name","last_name":"Last name"}
    }
}
```